packages/p2p/src/pages/my-profile/my-profile-stats/my-profile-name/my-profile-name.tsx

```diff
@@ -133,13 +133,25 @@ const MyProfileName = () => {
                                 <div className='my-profile-name__rating__row'>
                                     <BlockUserCount />
                                 </div>
+                                {/* TODO: uncomment when implementing business hours feature */}
```

Comment:
Can you explain this.

Reply:
currently we are not implementing the business hour feature just the UI, since this is a small part of it in my profile page, i had commented out the code and added a note to remove this once we are implementing the BE side of this
 
![Screenshot 2024-05-27 at 4 57 47 PM](https://github.com/binary-com/deriv-app/assets/103412909/224acc75-a02b-427f-b617-85585158c6c1)